.github/workflows/tests.yaml

```diff
@@ -0,0 +1,7 @@
+name: Run Tests
```

Comment:
I think we don't need this workflow since there are no unit tests for this plugin. This workflow runs the same tests as `acc-tests.yaml`